README.md

```diff
@@ -256,6 +256,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [First Citdael](http://www.skyislandsvr.com/pages/firstcitadel.html) - Citadel with flowing waterfalls
 - [Missed Connections](http://armthethinker.github.io/webVR-experiments/5--missed-connections.html) - Visualization of Craigslist "Missed Connections" section
 - [Aeon](http://aeon.horoman.com/) (by [@h0r0man]) - Aeon is a 3D environment with VR support, where the user runs an island that tells about the prequel to the main story of Aeon.
+- [SlideAmaze](http://slideamaze.com/) - Turns your slideshow presentation [into a 3D maze](http://fouryears.eu/2016/07/27/slide-a-maze/).
```

Comment:
Hey, thanks. If punctuation is important, this has to be noted in contributing.md. In this case I just followed the example of the line above, which had a dot at the end. Also, perhaps you could just go and fix all the punctuation to a unified format all over the file yourself.

As for https, I am totally aware of LetsEncrypt/StartSSL/CloudFare and other options, but this particular project was basically a one night's hack, hence optional features like "adding SSL" were out of scope. Secondly, this particular project is not meant as a place to handle secret information, hence protocol security is not one of its requirements in any sense. But next time I'm in procrastination mood perhaps I'll enable SSL as well, yes.